### ISSUE TITLE
Could not create split transaction

### ISSUE THREAD

**Issue by jkawamoto** [lukechampine/walrus-cli#11]
I got `Could not create split transaction: insufficient funds` when I tried to split an output.

It looks like balance calculation has a problem. It says it only has 5SC

```
$ ./walrus-cli -a http://localhost:9380/ balance
5SC
```

but I think it should have 5K SC

```
$ curl http://localhost:9380/utxos
[
        {
                "value": "5000000000000000000000000000",
                "unlockhash": "...",
                "ID": "..."
        }
]
```

**Comment by lukechampine** [comment]
This might be due to `balance` incorporating limbo transactions and `utxos` not. What do you see with `curl "http://localhost:9380/utxos?limbo=true"`?

https://github.com/lukechampine/walrus-cli/commit/c2577a1f011a29800c5679be84621d4c2260b6eb switches to using Limbo outputs everywhere, which might help.

**Comment by jkawamoto** [comment]
It looks the same:

```
$ curl "http://localhost:9380/utxos?limbo=true"
[
        {
                "value": "5000000000000000000000000000",
                "unlockhash": "...",
                "ID": "..."
        }
]
```

I've just created this wallet and sent 5K, so it shouldn't have limbo transactions.

This is the result of transactions command:

```
root@muse-0:~# ./walrus-cli -a http://localhost:9380/ transactions
Transaction ID             Height    Gain/Loss
...                       272067    -1624.289978597649134414662456SC
```

**Comment by lukechampine** [comment]
Ah, ok, I see what's going on. This should be an easy fix.

**Comment by lukechampine** [comment]
Fixed in 38365f6

**Comment by lukechampine** [comment]
oops, I guess this issue was also about the `insufficient funds` error, not just balance rendering. Please reopen if you still get `insufficient funds` (https://github.com/lukechampine/walrus-cli/commit/c2577a1f011a29800c5679be84621d4c2260b6eb may have helped...?)
